### PR TITLE
Have the CSV own both CRD's to fix SRVKS-396

### PR DESCRIPTION
**Pull request by jcrossley3**

### CODE REVIEW

File: olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -76,7 +76,7 @@ spec:
   customresourcedefinitions:
     owned:
     - description: Represents an installation of a particular version of Knative Serving
-      displayName: Knative Serving
+      displayName: Knative Serving 0.11.1
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml]: Should this just stay "Knative Serving"?

File: olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -89,6 +89,20 @@ spec:
         x-descriptors:
         - 'urn:alm:descriptor:io.kubernetes.conditions'
       version: v1alpha1
+    - description: Represents an installation of a particular version of Knative Serving
+      displayName: Knative Serving 0.10.0 (obsolete)
```
Comment (markusthoemmes) [inline olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml]:
```suggestion
      displayName: Knative Serving (obsolete group, see documentation)
```

maybe?

**Comment by mgencur** [comment]
Can you retrigger https://github.com/openshift-knative/serverless-operator/pull/80 when this is merged? It would be even better to include those tests here in this PR.

**Comment by mgencur** [comment]
Please see my comment on https://github.com/openshift-knative/serverless-operator/pull/80  where I included this commit.

File: olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (jcrossley3) [reply olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml]: I don't know. I was trying to include what I personally would expect to see if we have to show two of them. If I show the version of one, I know that would trigger in myself to know what the version of the other one is.

File: olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml (hunk 2 above)
Comment (jcrossley3) [reply olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml]: I think we need to sufficiently convey in their names why there need to be two. Ideally, we'd have some means of telling the console *NOT* to display one of our owned CRD's, which is something @bbrowning has asked for in the past.

File: olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml (hunk 1 above)
Comment (markusthoemmes) [reply olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml]: So as discussed in Slack yesterday, do we want to make this "Knative Serving" and the other "Knative Serving (deprecated)"? I feel like that sufficiently conveys that we want people to move to the "proper" one while the other is going away in the future.

Comment (mgencur) [reply olm-catalog/serverless-operator/1.4.0/serverless-operator.v1.4.0.clusterserviceversion.yaml]: Having the version there is confusing, IMO. I suppose people will think they can choose between https://github.com/knative/serving/releases/tag/v0.11.1 and https://github.com/knative/serving/releases/tag/v0.10.0 but they won't get Knative Serving of the given version, they will only get KnativeServing CRD of that version. And the version of Knative Serving won't be affected by the choice.

**Pull request closed without merging**